website/content/docs/deploy/service-mesh/enable.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: Enable service mesh
+description: >-
+    Learn how to enable and configure Consul's service mesh capabilities in agent configurations.
+---
+
+# Enable service mesh
+
+This page describes the process to enable Consul's service mesh features.
+
+For more information about configurable options in the service mesh and the process to full bootstrap Consul's service mesh, refer to [Connect services](/consul/docs/connect).
+
+## Enable mesh in server agent configuration
+
+Consul's service mesh features are not enabled by default. To enable the service mesh, you must change the configuration of your Consul servers. You do not need to change client agent configurations in order to use the service mesh.
+
+To enable Consul's service mesh, set `connect.enabled` to `true` in a new or existing [agent configuration file](/consul/docs/agent/config/config-files). On Kubernetes deployments, update the cluster's [Helm chart configuration](/consul/docs/k8s/helm) instead.
```

Comment:
```suggestion
Consul's service mesh features are not enabled by default when running Consul on virtual machines. To enable the service mesh, you must change the configuration of your Consul servers. You do not need to change client agent configurations in order to use the service mesh.

To enable Consul's service mesh, set `connect.enabled` to `true` in a new or existing [agent configuration file](/consul/docs/agent/config/config-files). Service mesh is enabled by default on Kubernetes deployment. Refer to the [Helm chart configuration](/consul/docs/k8s/helm) to verify that service mesh is enabled.
```
Ignore if this is incorrect, but I think SM is enabled by default on K8s, but not VMs.